README.md

```diff
@@ -36,19 +36,68 @@ scrape_configs:
     metrics_path: /export
     params:
       command: [check_load] # Run the check_load command.
+    relabel_configs:
+      - source_labels: [__address__]
+        target_label: __param_target
+      - source_labels: [__param_target]
+        target_label: instance
+      - target_label: __address__
+        replacement: 127.0.0.1:9275 # Nrpe exporter.
     static_configs:
       - targets: # Targets to run the specified command against.
         - '127.0.0.1:5666'
         - 'example.com:5666'
+
+  - job_name: nrpe_check_apt
+    honor_timestamps: true
```

Comment:
Why this?

Reply:
honor_timestamps is not needed - will remove.